test/setup.spec.js

```diff
@@ -355,4 +355,46 @@ describe('setup', () => {
       })
       .then(done);
   });
+
+  describe('Bindings', ()=>{
```

Comment:
If you really want to group them, something like `Methods` would be better
```suggestion
  describe('Methods', ()=>{
```